src/app/personside/infotabs/ytelser/Ytelser.tsx

```diff
@@ -35,27 +34,24 @@ const Styling = styled.section`
 `;
 
 function Ytelser() {
+    useKeepQueryParams();
     const ytelser = useBrukersYtelser();
     const dypLenker = useInfotabsDyplenker();
     const valgtYtelse = ytelser.ytelser.find(ytelse => dypLenker.ytelser.erValgt(ytelse));
-    const history = useHistory();
-
-    useEffect(() => {
-        if (!valgtYtelse) {
-            const førsteYtelse = ytelser.ytelser[0];
-            førsteYtelse && history.push(dypLenker.ytelser.link(førsteYtelse));
-        }
-    }, [ytelser.ytelser, dypLenker, history, valgtYtelse]);
 
     return (
         <Styling>
             {ytelser.feilmeldinger}
             <Layout>
                 <ScrollBar keepScrollId="ytelser-liste">
-                    <YtelseListe pending={ytelser.pending} ytelser={ytelser.ytelser} />
+                    <YtelseListe
+                        pending={ytelser.pending}
+                        ytelser={ytelser.ytelser}
+                        valgtYtelse={valgtYtelse || ytelser.ytelser[0]}
```

Comment:
Det er litt uheldig å løse det samme på to forskjellige steder, da blir det fort usynk ;-) Dette løses nok bedre ved å endre over slik:
```
    const valgtYtelse = ytelser.ytelser.find(ytelse => dypLenker.ytelser.erValgt(ytelse)) || ytelser.ytelser[0];

```